db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+Dependency.destroy_all
+Comment.destroy_all
+
+puts "create 2 dependencies: d1 and d2"
+d1 = Dependency.create(name: "Gem n°1")
```

Comment:
I'd recommend `Dependency#find_or_create_by!`, so that it's created only once,
and fail directly when the creation is not possible.